### DISCUSSION TITLE
Can Quickemu run macOS?

### DISCUSSION THREAD

**Discussion by blakemcbride** [quickemu-project/quickemu#1242] [Quickemu]
I use and love Linux nearly exclusively.  However, I sometimes need to test my software on other platforms, or on rare occasions, I have to run software that is exclusive to a certain platform.

Although QuickEMU is a great tool, running Linux or Windows on any VM platform is trivial.  But booting up macOS is very difficult.

Again, I like using QuickEMU for various Linux and Windows platforms, but at least for me, the biggest value of QuickEMU is running macOS.  (I use macOS for nothing other than testing ports of my own software!)

For me, and for whatever reason, running macOS with QuickEMU has been pretty flakey.  I got it working really well once.  I used it for some time but had to update the major revision of my OS (Fedora 38 → 40).  Since then, I have tried more than a half dozen times but haven’t been able to get macOS working.  Various Linux distros work fine with QucikEMU.

Although I like QuickEMU and appreciate its development, its primary value to me is in getting macOS to run on my Linux host.

I just wanted to share one user’s perspective.

Thank you for a much needed and beautiful tool!!

Blake McBride

**Comment by lj3954** [comment]
Please try the latest version of quickemu from git. PR https://github.com/quickemu-project/quickemu/pull/1225 includes numerous fixes to macOS guests. If this doesn't fix the issue, please post more information such as your config file, quickemu output, and details on where you're getting stuck in the installation.

**Comment by flexiondotorg** [comment]
Quickemu 4.9.5 is soon to be released and provides much improved compatibility of running macOS as a guest VM on Linux. This release also make it possible to run Quickemu on macOS hosts.
